src/Microsoft.Azure.WebJobs.Host/packages.config

```diff
@@ -18,7 +18,7 @@
   <package id="System.ComponentModel.EventBasedAsync" version="4.3.0" targetFramework="net45" />
   <package id="System.Console" version="4.3.0" targetFramework="net46" />
   <package id="System.Diagnostics.Debug" version="4.3.0" targetFramework="net46" />
-  <package id="System.Diagnostics.DiagnosticSource" version="4.3.0" targetFramework="net46" />
+  <package id="System.Diagnostics.DiagnosticSource" version="4.4.0" targetFramework="net46" />
```

Comment:
@fabiocav -- how do you feel about this consolidation? It'll happen anyway in Script since App Insights is referenced there... Just wanted to point it out.